src/docs/asciidoc/Sources.asciidoc

```diff
@@ -1118,7 +1118,7 @@ $$offsetUpdateCount$$:: $$frequency, in number of messages, with which offsets a
 $$offsetUpdateShutdownTimeout$$:: $$timeout for ensuring that all offsets have been written, on shutdown$$ *($$int$$, default: `2000`)*
 $$offsetUpdateTimeWindow$$:: $$frequency (in milliseconds) with which offsets are persisted mutually exclusive with the count-based offset update option (use 0 to disable either)$$ *($$int$$, default: `10000`)*
 $$partitions$$:: $$comma separated list of partition IDs to listen on$$ *($$String$$, default: ``)*
-$$queueSize$$:: $$the maximum number of messages held internally and waiting for processing, per concurrent handler. Value must be a power of 2.$$ *($$int$$, default: `1024`)*
+$$queueSize$$:: $$the maximum number of messages held internally and waiting for processing, per concurrent handler. Value must be a power of 2$$ *($$int$$, default: `1024`)*
```

Comment:
This is very strange; I didn't touch this file. Don't know why IDEA would mess with it. Feel free to exclude it from the merge.